draft-ietf-rats-corim.md

```diff
@@ -1491,36 +1491,97 @@ and how the verifier matches a Reference Value against the Accepted Claims Set
 
 #### Grouping Reference Values and Endorsements {#sec-grouping-ref-vals}
 
-> This paragraph will be replaced by a description of how the CoRIM tells the
-verifier which Reference Values and Endorsed Values are grouped together.
+> *TODO: This paragraph will be replaced by a description of how the CoRIM tells the
+verifier which Reference Values and Endorsed Values are grouped together.*
 
 [^issue]: Need to get agreement on how group membership is encoded. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/136
 
-[^issue]: Need to describe how to match conditional endorsements. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/80
+> *TODO: Can we find a better term than the placeholder "group"? If so then I think we should replace all mentions of "group" with the new term.*
 
 #### Matching all Reference Values in a group against the Accepted Claims Set {#sec-match-all-ref-vals}
 
-If all Reference Values in a group match entries in the Accepted Claims Set
-then all Endorsements in the group are added to the Accepted Claims Set
-(see {{sec-add-to-acs}}). {{sec-match-one-ref-val}} describes how one
+If a group contains at least one `reference-triple-record` and at least one
+`endorsed-triple-record` then the Reference Values contained in the former
+are used to decide whether to add Endorsed Values in the latter to the
+Accepted Claims Set.
+
+The Verifier SHALL try to match all Reference Values contained within all
+`reference-triple-record`s within the group against Evidence and Endorsements
+in the Accepted Claims Set. {{sec-match-one-ref-val}} describes how each
 Reference Value is matched against the Accepted Claims Set.
 
+If all Reference Values in a group match entries in the Accepted Claims Set
+then all Endorsed Values from `endorsed-triple-record`s in the group are added
+to the Accepted Claims Set (see {{sec-add-to-acs}}).
+
 If any Reference Value in a group does not match the Accepted Claims Set then
-all Endorsements in the group are silently ignored.
+all `endorsed-triple-record`s in the group are silently ignored.
+
+Each group is processed independently of other groups. If `reference-triple-record`s
+in a group do not match the Accepted Claims Set then this does not affect
+the processing of other groups.
+
+#### Matching a Conditional Endorsement Triple against the Accepted Claims Set
+
+A Conditional Endorsement Triple {{sec-comid-triple-cond-end}} is an efficient
+encoding of a Reference Value and an Endorsed Value.
```

Comment:
```suggestion
encoding of a Reference Value that describes the result of previously accepted Claims as the condition for accepting additional Endorsed Values.
```

Reply:
The next couple of lines say that, so I don't think it needs to say it here as well.